protocol/retrieval/api.go

```diff
@@ -0,0 +1,36 @@
+package retrieval
+
+import (
+	"context"
+	"io"
+
+	"gx/ipfs/QmR8BauakNcBa3RbE4nbQu76PDiJgoQgz8AJdhJuiU4TAw/go-cid"
+	"gx/ipfs/QmTu65MVbemtUxJEWgsTtzv9Zv9P8rvmqNA4eG9TrTRGYc/go-libp2p-peer"
+
+	"github.com/filecoin-project/go-filecoin/address"
+)
+
+// API here is the API for a retrieval client.
+type API struct {
+	rc       *Client
+	plumbing retrievalClientPorcelainAPI
```

Comment:
maybe call it porcelain?

Reply:
🤷‍♀️ it's called plumbing elsewhere, I guess because it's not meant to be used externally?

Reply:
probably a mistake, we originally depended o plumbing everywhere before we had porcelain, and there was probably a rename that missed unexported struct members